posthog/celery.py

```diff
@@ -312,7 +313,32 @@ def pg_plugin_server_query_timing():
             pass
 
 
-CLICKHOUSE_TABLES = ["events", "person", "person_distinct_id2", "session_recording_events"]
+POSTGRES_TABLES = ["posthog_personoverride", "posthog_personoverridemapping"]
+
+
+@app.task(ignore_result=True)
+def pg_row_count():
+    with pushed_metrics_registry("celery_pg_row_count") as registry:
+        row_count_gauge = Gauge(
+            "posthog_celery_pg_table_row_count",
+            "Number of rows per Postgres table.",
+            labelnames=["table"],
```

Comment:
It's personal preferences, so `nit`, but k8s environments add so many tags that I prefer longer-but-less-ambiguous tag names, such as `table_name` used by `clickhouse_lag`. Unfortunately, I've been inconsistent between PRs and `clickhouse_row_count` uses the shorter `table` tag name. Could you please align the three metrics to a unique name, one way or another?

Reply:
I do prefer longer-but-less-ambiguous names when possible, but opted to follow `clickhouse_row_count` convention for consistency (consistently inconsistent? :joy:)

I'll align everything with `clickhouse_lag`.